pom.xml

```diff
@@ -2324,13 +2414,13 @@
                 <plugin>
                     <groupId>org.apache.maven.plugins</groupId>
                     <artifactId>maven-surefire-plugin</artifactId>
-                    <!-- 3.0.0-M1 was released 2018-11-07. -->
-                    <version>3.0.0-M1</version>
+                    <!-- 3.0.0-M4 was released 2019-11-13 -->
+                    <version>3.0.0-M4</version>
```

Comment:
@jeffklassen NB: Upgrade to the Apache Maven surefire plugin.

Reply:
this was another attempt at fixing test stability, that didn't seem to actually affect anything. What finally worked was reducing the mini cluster tablet servers from 2 to 1.